Enterprise-CMCS/macpro-mdct-mfp
# Cmdct 3141 - prevent return key from exiting the modal and navigating back to prev page

### Description
Hitting `return` in the overlay modal type exited the modal and navigated back to the previous page.

https://github.com/Enterprise-CMCS/macpro-mdct-mfp/assets/22454493/8000f15f-f962-4d4a-ace8-f3f236fa08dc


 I changed the button `type` from `submit` to `button`, and the `return` key tabs to the next field instead of exiting the modal to fix this issue.


### Related ticket(s)


CMDCT-3141
---
### How to test
In `main`, go to the state or territory initiatives and try to add an evaluation plan or funding source--fill out one or two inputs and then hit `return`, see that you've been sent back to the previous page. 
In project branch do the same thing and see that the `return` acts like a `tab`, going to the next input section instead of exiting the modal (unless all required fields have been filled out, in which case the `return` will hit the `submit` button.


### Important updates




---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
